packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -75,7 +75,7 @@ typedef GestureVelocityTrackerBuilder = VelocityTracker Function(PointerEvent ev
 ///  * [HorizontalDragGestureRecognizer], for left and right drags.
 ///  * [VerticalDragGestureRecognizer], for up and down drags.
 ///  * [PanGestureRecognizer], for drags that are not locked to a single axis.
-abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
+sealed class DragGestureRecognizer extends OneSequenceGestureRecognizer {
```

Comment:
Looks like we did something similar in BaseTapAndDragGestureRecognizer. @Renzo-Olivares what do you think?

Reply:
I think this is safe to do. My initial though was that it might not be since `DragGestureRecognizer` has been around for a while, and sealing it now would break anyone who is extending it. Looking through github https://github.com/search?q=%22extends+DragGestureRecognizer+%7B%22+path%3A*.dart&type=code&p=1 this doesn't seem to be the case though. All the usages I could find at least where just forks of the framework. Internally I couldn't find anyone extending `DragGestureRecognizer` either.

Reply:
> My initial though was that it might not be since DragGestureRecognizer has been around for a while, and sealing it now would break anyone who is extending it

I don't think anyone is able to extend it right now, since there are private abstract methods that must be implemented.

Reply:
Now there's a real-world use case for https://github.com/dart-lang/linter/issues/4984!